### PR TITLE
Styling changes

### PR DESCRIPTION
**Pull request by ImCheesecake**
Fixed some styling issues on mobile/tablet. Made the side-menu a bit wider on desktop and added z-index on mobile.

### CODE REVIEW

File: apps/client/src/app/app.component.html [hunk 1]
```diff
@@ -202,6 +202,7 @@
                 *ngIf="!overlay"
                 [(nzCollapsed)]="settings.compactSidebar"
                 [nzCollapsedWidth]="80 | ifMobile: 0"
+                [nzWidth]="255"
```
Comment (Supamiu) [inline apps/client/src/app/app.component.html:205]:
```suggestion
                [nzWidth]="255 | ifMobile: 200"
```

I think it's better to keep it 200 on mobile, the ifMobile pipe being good for this kind of use case :)

File: apps/client/src/theme.less [hunk 2]
```diff
@@ -336,6 +336,7 @@ a, .ant-table-filter-dropdown-link {
 
 .ant-layout-sider {
   background: var(--topbar-color);
+
```
Comment (Supamiu) [inline apps/client/src/theme.less:339]:
```suggestion
```

**Review by Supamiu — CHANGES_REQUESTED** [review]
First static review without running it locally ^^

File: apps/client/src/app/app.component.html (hunk 1 above)
Comment (ImCheesecake) [reply apps/client/src/app/app.component.html:205]: Fair enough, I had that before but since I added the z-index I thought it could be wider :)

**Review by Supamiu — APPROVED** [review]

**Pull request merged**